data/built-ins.json

```diff
@@ -100,26 +100,31 @@
   },
   "es6.map": {
     "chrome": 51,
+    "firefox": 53,
```

Comment:
Every single feature and observable thing is significant - if something isn't 100% compliant, it's broken, and it needs to be patched/polyfilled/worked around.